resources/lib/person.py

```diff
@@ -131,11 +151,14 @@ def get_tvshow_list(self):
                 list_item, is_local = tmdb_handle_tvshow(item,self.local_shows)
                 li.append(list_item)
                 duplicate_handler.append(item['id'])
+                item['type'] = 'tvshow'
+                item['release_date'] = item['first_air_date']
```

Comment:
`first_air_date` is copied into a new key `release_date`, meaning the movie and tvshow lists can be concatenated and the dict sorting by `release_date` them works perfectly.